### PR TITLE
Reorganize performance suites to avoid latest singleton

### PR DESCRIPTION
**Pull request by lydia-duncan**
This
- removes the top-level suites "Chapel versus C comparisons", "Statement
  Comparisons" and "Math operations"
- creates a new top-level suite "Serial Performance" and two sub-suites,
  "Chapel versus C Comparisons" and "Statment Comparisons"
  - moves entirety of old top level suite "Chapel versus C comparisons" into
    suite of same name, and moves entirety of "Math operations" into the same
    suite
- moves some of "Statement Comparisons" into a new suite, "Parallel Statement
  Comparisons"
  - moves the rest of the old "Statement Comparisons" suite into the new
    sub-suite of the same name.

Verified output with --generate-graphs and that no .graph was missing after the
commit.

### CODE REVIEW

File: test/GRAPHFILES [hunk 1]
```diff
@@ -288,7 +274,37 @@ users/npadmana/twopt/twopt-buildtrees.graph
 users/npadmana/twopt/twopt-paircount.graph
 # suite: Elegance
 studies/elegance/elegance.graph
-# suite: Math operations
+# suite: Scalar Performance
```
Comment (ben-albrecht) [inline test/GRAPHFILES]: - Would 'serial performance' be a more fitting title?

Comment (lydia-duncan) [inline test/GRAPHFILES]: Ugh, we totally said that and for some reason I wrote down "scalar" . . .

Comment (ben-albrecht) [inline test/GRAPHFILES]: You had it right: "scalar" was said in the meeting, but I think "serial" is a better title.

**Comment by ben-albrecht** [comment]
Oh - be sure to update PR description

**Pull request merged**